addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/Element.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2020 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.openapi.generators;
+
+public class Element {
+
+    public enum Json {
```

Comment:
This and `Form` could override `toString()` (instead of providing `syntax()`), less verbose when concatenating them.

Reply:
Ah ha! The verbosity thing was what stopped me to create the enum in the first place. I wish I had thought about the toString method before. Good catch =)